packages/amplication-client/src/Entity/EntityField.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useCallback, useMemo, useContext, useState } from "react";
+import { useCallback, useMemo, useContext, useState } from "react";
```

Comment:
React suppose to be at any component file

Reply:
I think React import is no longer needed from React v17 (we are now at v18+) @barshimi